src/components/Slider/Slider.tsx

```diff
@@ -1,159 +1,376 @@
 
     return (
-        <div className="tw-flex">
-            <fieldset
-                {...radioGroupProps}
-                data-test-id="slider"
-                className="tw-relative tw-h-9 tw-w-full tw-grid tw-grid-flow-col tw-auto-cols-fr tw-justify-evenly tw-p-0 tw-border tw-border-black-20 tw-m-0 tw-bg-black-0 tw-rounded tw-font-sans tw-text-s tw-select-none"
+        <div className="tw-flex tw-flex-col" data-test-id="fondue-slider" id={id}>
+            <label
+                htmlFor={id}
+                className={merge([!label && 'tw-hidden', disabled && 'tw-text-text-disabled'])}
+                data-test-id="fondue-slider-label"
             >
-                <motion.div
-                    aria-hidden="true"
-                    // div border is not included in width so it must be subtracted from translation.
-                    animate={{ x: `calc(${100 * selectedIndex}% - ${2 * selectedIndex}px)` }}
-                    initial={false}
-                    transition={{ type: 'tween', duration: 0.3 }}
-                    style={{
-                        width: `${100 / items.length}%`,
-                    }}
-                    hidden={!activeItemId}
-                    className={merge([
-                        'tw-absolute tw--inset-px tw-h-full tw-box-content tw-border tw-rounded tw-pointer-events-none',
-                        disabled
-                            ? 'tw-border-line-x-strong tw-border-opacity-30 tw-bg-black-0'
-                            : 'tw-border-black tw-bg-white',
-                    ])}
-                />
-                {itemElements}
-            </fieldset>
+                {label}
+            </label>
+            <div className="tw-flex">
+                <div className={merge(['tw-flex-1 tw-flex tw-items-center'])}>
+                    {showMinMax && (
+                        <div className={merge(['tw-mr-3', disabled && 'tw-text-text-disabled'])}>
+                            {min}
+                            {valueSuffix}
+                        </div>
+                    )}
+                    <button
+                        ref={setSliderRef}
+                        data-test-id="fondue-slider-interactive"
+                        className="tw-flex-1 tw-relative tw-h-full tw-cursor-pointer disabled:tw-cursor-default"
+                        onMouseOver={onMouseOver}
+                        onMouseOut={onMouseOut}
+                        onFocus={onFocus}
+                        onBlur={onBlur}
+                        onMouseDown={startDrag}
+                        onMouseUp={stopDrag}
+                        onTouchStart={startDrag}
+                        onTouchEnd={stopDrag}
+                        onKeyDown={onKeyDown}
+                        disabled={disabled}
+                    >
+                        <span className="tw-absolute tw-block tw-top-1/2 tw--translate-y-1/2 tw-w-full tw-h-1 tw-rounded-sm tw-bg-box-neutral tw-flex-1"></span>
```

Comment:
@fabriciomendonca I just have a question you can surely clarify. What was the reason to better use span and button instead of input type=range as starting point?

Reply:
I used as a base the HTML structure of the Carbon `Slider` component and the recommendations for accessible slider with `aria-` properties, instead of using the input type=range and keeping it hidden.
I decided for the `button` because it is focusable and tabbable by default, instead of using a div and adding the `tabIndex=-1`. Actually, I've tried it and it was not focusing the element properly. :)
For reference, the Carbon Slider HTML structure
```html
<div class="cds--form-item">
  <label for="slider" class="cds--label" id="slider-label">Slider label</label>
  <div class="cds--slider-container">
    <span class="cds--slider__range-label">0</span>
    <div class="cds--slider" role="presentation" tabindex="-1" stepmuliplier="1" value="50">
      <div class="cds--slider__thumb" role="slider" id="slider" tabindex="0" aria-valuemax="100" aria-valuemin="0" aria-valuenow="34" aria-labelledby="slider-label" style="left: 34%;"></div>
      <div class="cds--slider__track"></div>
      <div class="cds--slider__filled-track" style="transform: translate(0%, -50%) scaleX(0.34);"></div>
    </div>
    <span class="cds--slider__range-label">100</span>
    <input type="number" id="slider-input-for-slider" class="cds--text-input cds--slider-text-input" aria-label="Label for slider value" min="0" max="100" step="1" value="34">
  </div>
</div>
```